tests/test_autodoc.py

```diff
@@ -1341,6 +1341,19 @@ def test_descriptor_class(app):
     ]
 
 
+@pytest.mark.sphinx('html', testroot='ext-autodoc')
+def test_autofunction_for_callable(app):
+    actual = do_autodoc(app, 'function', 'target.callable.function')
+    assert list(actual) == [
+        '',
+        '.. py:function:: function(arg1, arg2, **kwargs)',
+        '   :module: target.callable',
+        '',
+        '   A callable object behaves like ',
```

Comment:
I dunno if you want grammar corrections on these things, but this sentence fragment doesn't make much sense to me. Should it be, "A callable object that behaves like a function"?

Reply:
Oh, thanks. very helpful for me!